public/js/home.js

```diff
@@ -1264,6 +1264,31 @@ document.getElementById("default_open").click();
 
 //#ifndef lite
 $.ajax("/version").then(ver => $("#version").text(ver));
+
+$.ajax("/updates").then(upt => {
+	$("#updates").html(upt);
+	document.getElementById("changelog").outerHTML = "<h2 class='info-header'>Version History/What's New:</h2>";
+    var items = document.getElementById("updates").getElementsByTagName("h2");
+	for (var i = 0; i < items.length; i++) {
+	    items[i].className = "info-header";
+	}
```

Comment:
Instead of using a C-style `for` loop prone to off-by-one errors, this can be done more simply:
```suggestion
	items.forEach(x => { x.className = "info-header"; });
```